automated-comments/csharp/general/use_string_interpolation_not_string_format.md

```diff
@@ -0,0 +1,3 @@
+Rather than using `string.Format` to dynamically build a string, the more common C# approach is to use [string interpolation](https://csharp.net-tutorials.com/operators/the-string-interpolation-operator/).
```

Comment:
Perhaps also invert this?

```suggestion
A more common approach in C# is to use [string interpolation](https://csharp.net-tutorials.com/operators/the-string-interpolation-operator/), rather than using `string.Format` to dynamically build a string
```